explainaboard/processors/processor_registry.py

```diff
@@ -3,9 +3,89 @@
 from __future__ import annotations
 
 from explainaboard import TaskType
+from explainaboard.processors.argument_pair_extraction import (
```

Comment:
This file looks no longer so-called registry (in our terms), so the file name should be changed.

Reply:
Yeah, renamed it to `processor_factory.py`